docker/build_scripts/build.sh

```diff
@@ -72,6 +72,8 @@ yum -y erase wireless-tools gtk2 libX11 hicolor-icon-theme \
     avahi freetype bitstream-vera-fonts \
     ${PYTHON_COMPILE_DEPS}  > /dev/null 2>&1
 yum -y install ${MANYLINUX1_DEPS}
+# Add a soft-link from cmake28 to cmake
+ln -s `which cmake28` /usr/local/bin/cmake
```

Comment:
Perhaps, the system one is pretty ancient though (2.6.4), so I sort of expect that it would be fine.  At the moment I can't even get cmake28 to install (argh).  It was working yesterday.  I don't understand what is going on, something to do with missing libraries.